paperdraft.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM pandoc/latex:2.9.2.1 AS papergen
+
+# Install additional LaTeX packages
+RUN tlmgr install \
```

Comment:
@tarleb - not sure if you've seen this list already but here are the packages Whedon currently has available: https://github.com/openjournals/heroku-buildpack-tex/blob/master/bin/compile#L85-L109

Reply:
Thanks, I hadn't seen that yet. The packages are hard-coded for now, maybe we can find a way to share the list (or the image).